src/containers/Toolbar/index.tsx

```diff
@@ -10,19 +11,34 @@ interface IToolbarProps {
   onFilterValueChange: (filterValue: string) => void;
   preserveLogs: boolean;
   onPreserveLogsChange: (preserveLogs: boolean) => void;
+  onClear: () => void;
 }
 
+const ClearButton = ({ onClick }: { onClick: () => void }) => (
+  <div className="opacity-50 outline-none hover:opacity-100">
```

Comment:
Almost there with this one. Last thing is these classes are not needed `opacity-50 outline-none hover:opacity-100`. Opacity is already handled within the IconButton component and outline is needed for accessibility.

Reply:
What do you think about to just remove the component Clear button and use the IconButton directly inside the Toolbar?

Reply:
Yeah that seems sensible.